webapp/channels/src/utils/performance_telemetry/reporter.ts

```diff
@@ -2,8 +2,8 @@
 // See LICENSE.txt for license information.
 
 import type {Store} from 'redux';
-import {onCLS, onFCP, onINP, onLCP, onTTFB} from 'web-vitals';
-import type {Metric} from 'web-vitals';
+import {onCLS, onFCP, onINP, onLCP, onTTFB} from 'web-vitals/attribution';
+import type {INPMetricWithAttribution, LCPMetricWithAttribution, Metric} from 'web-vitals/attribution';
```

Comment:
The attribution version of this library is a bit bigger than the regular one, but it's only about 600 bytes when compressed according to its docs